lib/src/main/java/com/auth0/jwt/JWTVerifier.java

```diff
@@ -51,7 +54,7 @@ static Verification init(Algorithm algorithm) throws IllegalArgumentException {
             }
 
             this.algorithm = algorithm;
-            this.claims = new HashMap<>();
+            this.claims = new HashMap<>(48);
```

Comment:
No magic numbers please. Can we move it into a constant and give it a proper name/description?

Reply:
On second thought, 16 (the default) is probably okey here.

Reply:
Agree. I've never seen a token with that many claims 😆